### DISCUSSION TITLE
如何适配1080P以下的屏幕

### DISCUSSION THREAD

**Discussion by pneuma124** [NateScarlet/auto-derby#123] [Q&A] [answered]
作者您好，这是我第一次使用github，想请问一下如何将脚本适配至1366*768的电脑屏幕上？
我自己尝试修改了dmm.py中height和width的数值，但仅仅修改了窗口大小，后续判定无法运行。
本人代码能力不太行，就想请问下是否需要修改所有选项键位坐标才能适配至低分辨率的电脑上？
（如果是这样的话我还是考虑换电脑比较快一些。）

**Comment by NateScarlet** [comment, marked as answer]
分辨率太小的话 OCR 识别精度不够
你可以尝试把屏幕方向改成竖向再用 笔记本的话很可能用 Ctrl+Alt+方向键可以改

**Reply by NateScarlet** [reply]
所有坐标都是相对当前宽度计算的 只要是 9:16 的游戏画面理论上都不用改 不过OCR数据基本需要重新标

**Reply by pneuma124** [reply]
> 分辨率太小的话 OCR 识别精度不够
> 你可以尝试把屏幕方向改成竖向再用 笔记本的话很可能用 Ctrl+Alt+方向键可以改

感谢回复，确实用了这个方法就可以了。
